lib/presentation/molecules/study_type_selection_molecule.dart

```diff
@@ -40,23 +40,29 @@ class _StudyTypeSelectionMoleculeState
   Widget build(BuildContext context) {
     return Column(
       children: [
-        ListTileAtom(
-          StudyType.analytically.previewName,
-          Radio<StudyType>(
-            value: StudyType.analytically,
-            groupValue: selectedType,
-            onChanged: onChanged,
+        InkWell(
```

Comment:
Please create a new radio button function that will include the text as well.
Also make sure that clicking on InkWell area also checks the radio button button and not just calling the method.